### DISCUSSION TITLE
Subtitles

### DISCUSSION THREAD

**Discussion by satstarter** [staxrip/staxrip#898] [General]
Hi, i am new here i love staxrip but i have a problem and i cant find the solution
the following is my problem convert a mkv to mp4 with 2 subtitles one is srt the other is ass
both in the same language i cant find howto add auto these 2 subtitles
it only gives me one subtitle or none ?
thanx in advance if you could help me

**Comment by satstarter** [comment]
ok figured it out sorry i did'nt know mp4 dont accept ass subtitles 
thanx

**Comment by jessielw** [comment]
The mp4 spec for subtitles technically doesn't support more than 1 subtitle either
